client/ayon_blender/plugins/create/create_render.py

```diff
@@ -1,53 +1,107 @@
 """Create render."""
 import bpy
+from typing import Optional
 
-from ayon_core.pipeline.context_tools import version_up_current_workfile
-from ayon_blender.api import plugin, lib
-from ayon_blender.api.render_lib import prepare_rendering
-from ayon_blender.api.workio import save_file
+from ayon_core.pipeline.create import CreatedInstance
+from ayon_blender.api import plugin, lib, prepare_rendering
 
 
-class CreateRenderlayer(plugin.BlenderCreator):
-    """Single baked camera."""
+class CreateRender(plugin.BlenderCreator):
+    """Create render instance."""
 
     identifier = "io.openpype.creators.blender.render"
     label = "Render"
-    product_type = "renderlayer"
+    product_type = "render"
     icon = "eye"
 
+    # TODO: Convert legacy instances to new style instances by finding the
+    #  relevant file output node and moving the imprinted data there.
+
+    def _find_existing_compositor_output_node(self) -> Optional["bpy.types.CompositorNodeOutputFile"]:
+        if not bpy.context.scene.use_nodes:
+            return None
+
+        # TODO: If user has a selected compositor node, prefer that one
```

Comment:
```suggestion
        # TODO: If user has a selected compositor node, prefer that one
```
Should we do it now instead of just forcing one particular node?